morty/analytical/lineshapes.py

```diff
@@ -42,6 +42,9 @@ def pseudovoigt(x_axis, iso, sigma, gamma, intensity, eta):
         Pseudo Voigt Profile for given params of same length as s_axis.
 
     """
```

Comment:
documentation for parameter gamma was wrong, the function raised an error if gamma = None was passed. If gamma is None it is now set to the value of sigma to reflect the description given.